### PR TITLE
Fix storage and object_count calculations on console

### PR DESCRIPTION
**Pull request by rikysya**
What:  fix storage and object values for buckets on console. Instead of storage*hours and objects*hours as we have in rollups, show the current bucket size and object count

https://storjlabs.atlassian.net/browse/V3-1791 there is still this issue so displayed values won't be correct, until it fixed

Why: https://storjlabs.atlassian.net/browse/V3-1866

Thanks for submitting a PR!

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Could the PR be broken into smaller PRs?
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?

### CODE REVIEW

File: satellite/satellitedb/usagerollups.go [hunk 1]
```diff
@@ -293,29 +298,24 @@ func (db *usagerollups) GetBucketTotals(ctx context.Context, projectID uuid.UUID
...
+		var objectCount uint64
+		err = storageRow.Scan(&inline, &remote, &objectCount)
+		if err != nil {
+			if err != sql.ErrNoRows {
+				return nil, err
+			}
 		}
 
+		// fill storage and object count
+		bucketUsage.Storage = memory.Size(inline + remote).GB()
```
Comment (mobyvb) [inline satellite/satellitedb/usagerollups.go]: should this still be done if `err == sql.ErrNoRows`?

File: satellite/console/usagerollups.go [hunk 2]
```diff
@@ -35,7 +35,7 @@ type BucketUsage struct {
 
 	Storage     float64
 	Egress      float64
-	ObjectCount float64
+	ObjectCount uint64
```
Comment (VinozzZ) [inline satellite/console/usagerollups.go]: Not sure if this applies to this convention but we seem to prefer using `int` over unsigned integer https://github.com/storj/docs/blob/master/code/Style.md#integers

File: satellite/satellitedb/usagerollups.go (hunk 1 above)
Comment (rikysya) [reply satellite/satellitedb/usagerollups.go]: yeah, it will return zero

File: satellite/console/usagerollups.go (hunk 2 above)
Comment (rikysya) [reply satellite/console/usagerollups.go]: seems, you are right

**Pull request merged**